src/lighteval/models/vllm/vllm_model.py

```diff
@@ -318,6 +318,7 @@ def _generate(
         generate: bool = True,
     ) -> list[GenerativeResponse]:
         """Contains the actual logic of the generation."""
+        print(f"{self.sampling_params.clone()=}")
```

Comment:
If you want to keep the prints for debug, maybe use the logger

Reply:
Ah good catch, sorry for including that!